components/builder-graph/src/main.rs

```diff
@@ -18,6 +18,8 @@
 #[macro_use]
 extern crate bitflags;
 #[macro_use]
+extern crate clap;
```

Comment:
We should use the Rust 2018 macro import syntax here: 

https://doc.rust-lang.org/edition-guide/rust-2018/macros/macro-changes.html

Specifically we can drop the `#[macro_use]; extern crate clap;` and add `clap_app` to the `use clap::` below;

The remainder of the `#[macro_use]` instance can be cleaned up similarly, but as it's a bit of a yak shave that might be good to make its own PR.